dali/pipeline/operator/op_schema.h

```diff
@@ -768,7 +826,7 @@ class SchemaRegistry {
 };
 
 template<typename T>
-inline T OpSchema::GetDefaultValueForOptionalArgument(const std::string &s) const {
+inline T OpSchema::GetDefaultValueForArgument(const std::string &s) const {
   const Value *v = FindDefaultValue(s, false, true).second;
   DALI_ENFORCE(v != nullptr, "Optional argument \"" + s + "\" is not defined for schema \""
```

Comment:
Shouldn't it say now: `make_string("The argument \"", name, "\" doesn't have a default value in schema \"", name(), "\"")` ?

Reply:
:+1: